test/controllers/call_lists_controller_test.rb

```diff
@@ -4,11 +4,12 @@ class CallListsControllerTest < ActionDispatch::IntegrationTest
   before do
     @user = create :user, role: 'admin'
     @user_2 = create :user, role: 'cm', name: 'Billy Everyteen'
-    @patient_1 = create :patient, name: 'Susan Everyteen'
-    @patient_2 = create :patient, name: 'Yolo Goat'
+    @line = create :line
```

Comment:
most of the test file changes look like this, in some way/shape/form. Most of the tests continue to work more or less as is, thanks to rails magic around foreign keys.